.secrets.baseline

```diff
@@ -449,5 +449,5 @@
       }
     ]
   },
-  "generated_at": "2024-11-27T14:06:13Z"
+  "generated_at": "2024-12-11T14:29:26Z"
```

Comment:
(opinionated) nitpick: It would make more sense to me to make the **secrets** corrections together with the changes they caused them so it is clear why we do them. It may be tedious when there are many commits (which is not the case here) and I sure there are different approaches to this in our team so no strong preference :smile_cat: